javaparser-testing/src/test/resources/com/github/javaparser/bdd/comment_parsing_scenarios.story

```diff
@@ -54,11 +54,11 @@ Then block comment 4 is " comment put randomly in class:    another orphan.    I
 Then block comment 5 is " a comment lost inside a compilation unit. It is orphan, I am sure you got this one  "
 Then the block comments have the following positions:
 |beginLine|beginColumn|endLine|endColumn|
-|3|1|3|82|
-|4|1|4|25|
-|7|5|7|30|
-|10|5|13|32|
-|17|1|17|89|
+|3|1|3|81|
```

Comment:
This is also a nice one. I could have changed the code to match the tests, but I checked it by hand and the end of the range was pointing one character after the end. The old parser was buggy. So I would say it would cause users to not have to work around this.